CanvasPlusPlayground/Features/Navigation/Sidebar.swift

```diff
@@ -58,51 +66,83 @@ struct Sidebar: View {
         .navigationTitle("Home")
         #if os(macOS)
         .navigationSplitViewColumnWidth(min: 275, ideal: 275)
-        #endif
-        .listStyle(.sidebar)
-        #if os(iOS)
         .toolbar {
-            ToolbarItem(placement: .cancellationAction) {
-                Button("Settings", systemImage: "gear") {
-                    navigationModel.showSettingsSheet.toggle()
+            ToolbarItem(placement: .confirmationAction) {
+                if let currentUser = profileManager.currentUser {
+                    Button {
+                        navigationModel.showProfileSheet.toggle()
+                    } label: {
+                        ProfilePicture(user: currentUser, size: 19)
+                    }
                 }
             }
         }
```

Comment:
To maintain the SwiftUI navigation logic, we keep ".navigationTitle("Home") as an inline style. However, we hide that style with a custom, empty toolbar item